packages/portal-proto/src/features/facets/EnumFacet.tsx

```diff
@@ -442,6 +442,7 @@ const EnumFacet: React.FC<FacetCardProps<EnumFacetHooks>> = ({
                           >
                             <div className="flex-none">
                               <Checkbox
+                                aria-label={`Select the ${field} facet`}
```

Comment:
what this this labeling? this is a loading view and should be disabled?

Reply:
removed it since its disabled/loading view. good catch